packages/composites/proof-of-reserves/README.md

```diff
@@ -42,12 +42,14 @@ See the [Composite Adapter README](../README.md) for more information on how to
-|           |   `addresses`   |           An array of addresses to get the balance from, when "protocol" is set to `list`            |                                                                                                                                                                         |             |
+| Required? |                Name                |                                             Description                                              |                                                                                 Options                                                                                 | Defaults to |
+| :-------: | :--------------------------------: | :--------------------------------------------------------------------------------------------------: | :---------------------------------------------------------------------------------------------------------------------------------------------------------------------: | :---------: |
+|    ✅     |             `protocol`             |                                 The protocol external adapter to use                                 |                                      `celsius_address_list`, `chain_reserve_wallet`, `gemini`, `list`, `renvm`, `wbtc`, `wrapped`                                       |             |
+|    ✅     |             `indexer`              |                                 The indexer external adapter to use                                  | `ada_balance`, `amberdata`, `bitcoin_json_rpc`, `blockchain_com`. `blockchair`, `blockcypher`,`btc_com`, `cryptoapis`, `eth_balance`, `lotus`, `por_indexer`, `sochain` |             |
+|           |          `confirmations`           | The number of confirmations required for a transaction to be counted when getting an address balance |                                                                                                                                                                         |      6      |
+|           |            `addresses`             |           An array of addresses to get the balance from, when "protocol" is set to `list`            |                                                                                                                                                                         |             |
+|           |     `disableAddressValidation`     |               if this parameter is set to `true` address validation will be turned off               |                                                                             `true`, `false`                                                                             |   `false`   |
```

Comment:
When would one want to use this feature?

Reply:
Great question.  Please ask Jonas because I 100% agree.  I see no reason to ever disable these options.

Reply:
@boxhock